contexts/ConfigContext.tsx

```diff
@@ -8,26 +9,33 @@ interface RuntimeConfig {
   flowAccessNode: string
 }
 
+const {publicRuntimeConfig} = getConfig()
+
 const defaultConfig = {
-  flowAccountAddress: "",
-  flowAccountPrivateKey: "",
-  flowAccountPublicKey: "",
-  flowAccountKeyId: "",
-  flowAccessNode: "",
+  flowAccountAddress: publicRuntimeConfig.flowAccountAddress || "",
+  flowAccountPrivateKey: publicRuntimeConfig.flowAccountPrivateKey || "",
+  flowAccountPublicKey: publicRuntimeConfig.flowAccountPublicKey || "",
+  flowAccountKeyId: publicRuntimeConfig.flowAccountKeyId || "",
+  flowAccessNode: publicRuntimeConfig.flowAccessNode || "",
 }
 
 export const ConfigContext = createContext<RuntimeConfig>(defaultConfig)
 
 export async function fetchConfigFromAPI(): Promise<RuntimeConfig> {
+  if (publicRuntimeConfig.isLocal) {
+    return defaultConfig
+  }
```

Comment:
Nice. Just to confirm, does the app still work if no local config is passed?

Reply:
I'm just wondering if the import of `next/config` does anything weird at runtime if there is no config

Reply:
hmm, this may be related to an issue we're seeing 
cc @justinbarry

Reply:
I just pushed `.env.development` for local dev.